dora/tests/src/test/java/alluxio/server/ft/MultiWorkerIntegrationTest.java

```diff
@@ -9,219 +9,219 @@
  * See the NOTICE file distributed with this work for information regarding copyright ownership.
  */
 
-package alluxio.server.ft;
-
-import static org.junit.Assert.assertEquals;
-
-import alluxio.AlluxioURI;
-import alluxio.Constants;
-import alluxio.annotation.dora.DoraTestTodoItem;
-import alluxio.client.WriteType;
-import alluxio.client.block.BlockStoreClient;
-import alluxio.client.block.BlockWorkerInfo;
-import alluxio.client.block.policy.BlockLocationPolicy;
-import alluxio.client.block.policy.options.GetWorkerOptions;
-import alluxio.client.file.FileInStream;
-import alluxio.client.file.FileSystem;
-import alluxio.client.file.FileSystemContext;
-import alluxio.client.file.FileSystemTestUtils;
-import alluxio.client.file.URIStatus;
-import alluxio.client.file.options.InStreamOptions;
-import alluxio.client.file.options.OutStreamOptions;
-import alluxio.conf.AlluxioConfiguration;
-import alluxio.conf.Configuration;
-import alluxio.conf.PropertyKey;
-import alluxio.grpc.CreateFilePOptions;
-import alluxio.grpc.OpenFilePOptions;
-import alluxio.grpc.WritePType;
-import alluxio.testutils.BaseIntegrationTest;
-import alluxio.testutils.LocalAlluxioClusterResource;
-import alluxio.util.io.BufferUtils;
-import alluxio.wire.BlockInfo;
-import alluxio.wire.FileBlockInfo;
-import alluxio.wire.WorkerNetAddress;
-
-import com.google.common.io.ByteStreams;
-import org.apache.commons.io.IOUtils;
-import org.junit.Assert;
-import org.junit.Ignore;
-import org.junit.Rule;
-import org.junit.Test;
-
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.OutputStream;
-import java.util.List;
-import java.util.Optional;
-import java.util.stream.StreamSupport;
-
-/**
- * Tests a cluster containing multiple workers.
- */
-@Ignore
-@DoraTestTodoItem(action = DoraTestTodoItem.Action.FIX, owner = "bowen",
-    comment = "fix the tests")
-public final class MultiWorkerIntegrationTest extends BaseIntegrationTest {
```

Comment:
@JiamingMai @dbw9580  See I removed this integration test here, but I think you wanna take a look at the test cases. We should create a new integration test based on top of dora and the test cases here can be applicable.

Reply:
Yes. We need to create a new one. The old integration test should be removed.